code/modules/reagents/Chemistry-Readme.dm

```diff
@@ -1,324 +0,0 @@
-/*
```

Comment:
Is this a todo? If you're removing the file then delete the file.

Reply:
Iiiiiiiiiiiiiiiiiiii thought I deleted the file.